modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -36,6 +36,21 @@ a|
 
 The Cypher query option `connectComponentsPlanner` is deprecated and will be removed without a replacement.
 The product's default behavior of using a cost-based IDP search algorithm when combining sub-plans will be kept.
+
+a|
+//not sure what category this should be, it is more information about a coming breaking change than actual deprecation
+label:returnValues[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+SHOW NODE UNIQUENESS CONSTRAINTS YIELD type
+----
+a|
+
+The current constraint type for node property uniqueness constraints, `UNIQUENESS`, will be updated to `NODE_UNIQUENESS` in Neo4j version 6.0.
+
+This will also be reflected in updates to some error messages and query statistics.
+
```

Comment:
Im not sure we need to advertise this on this page until it actually happens. I think your notes on the actual constraints page will suffice for now.

Reply:
If you think so, I don't feel that strongly and had a hard time trying to decide how to formulate it anyway 🤷